src/Bicep.Core/Parsing/Parser.cs

```diff
@@ -286,28 +286,32 @@ private SyntaxBase ModuleDeclaration(IEnumerable<SyntaxBase> leadingNodes)
         private ImportDeclarationSyntax ImportDeclaration(IEnumerable<SyntaxBase> leadingNodes)
         {
             var keyword = ExpectKeyword(LanguageConstants.ImportKeyword);
```

Comment:
Please also update:
* https://github.com/Azure/bicep/blob/main/docs/grammar.md
* https://github.com/Azure/bicep/blob/main/src/highlightjs/src/bicep.ts
* https://github.com/Azure/bicep/blob/main/src/monarch/src/bicep.ts
* https://github.com/Azure/bicep/blob/main/src/textmate/src/bicep.ts

Reply:
Updated the grammar files.